apiary.apib

```diff
@@ -762,6 +762,39 @@ Get a list of all work types, sorted alphabetically (on their name).
                 + id: `811a5825-96f4-4318-83c3-2840935c6003` (string)
                 + name: `Planning` (string)
 
+## Templates [/templates]
+
+### templates.list [POST /templates.list]
+
+Get a list of all templates.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + department_id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+            + type: `invoice` (enum[string])
+                + Members
+                    + delivery_note
+                    + invoice
+                    + order
+                    + order_confirmation
+                    + quotation
+                    + timetracking_report
+                    + workorder
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `a344c251-2494-0013-b433-ccee8e8435e5` (string)
+                + department (object)
+                    + type: `department` (string)
+                    + id: `36386b05-936e-4cc0-9523-bd20d797ebf5` (string)
+                + type: `invoice` (string)
+                + is_default: true (boolean)
```

Comment:
Q: `default` should we remove the `is`? looking at the invoicing endpoint we use `paid` and not `is_paid` we also use `invoicable` instead of `is_invoicable`

Reply:
Valid point. @dietrichm, @jenssegers Should/can I change this?

Reply:
I don't agree. For Paid, you can ask `Paid? Yes/No` ... Same for `Invoicable? Yes/No?` ... But if you try to do the same for `default?` you will automatically as for `is Default? Yes/No`

Reply:
Indeed, I assume if I see a `default` key, I'll get an identifier or object that is the default.
But I get what @mrtus is saying.

Reply:
For me it can stay as `is_default`.

Reply:
Is it paid, is it invoicable, is it default? That is what I would ask @rathesDot but sure if we decide this is correct for this use-case we can leave as is.

Reply:
I was also a bit doubting about the `is_` to be honest. I'm also ok with just having `default`, but no real preference to one.

Reply:
Okay, I'll leave it `is_default`.